app/components/user_item/index.ts

```diff
@@ -28,4 +28,4 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
     };
 });
 
-export default withDatabase(enhanced(AtMentionItem));
+export default withDatabase(enhanced(UserItem));
```

Comment:
As we discussed, we should refactor the withObservables HOC to the new way.

Reply:
I don't know what you mean here, is already using the new queries observables

Reply:
We could use the `observeConfigBooleanValue` (or something like that), but would fetch the config from the database twice. The cache should handle that without any problem, but stil... 0/5.

Reply:
this is just piping not really querying, so I don't see a reason to do that, also in the common queries PR that was something that I considered to provide as feedback but this way makes more sense when you need multiple values.